### PR TITLE
Remove all cart items at once

### PR DESCRIPTION
**Pull request by ILDaviz**

### CODE REVIEW

**Review by ghermans — CHANGES_REQUESTED** [review]
@ILDaviz please also add the translations for the other languages

**Comment by Sanjaybhattwebkul** [comment]
@ILDaviz 
There should be a **confirmation box before deleting** all items from the cart.

**Comment by ILDaviz** [comment]
> @ILDaviz There should be a **confirmation box before deleting** all items from the cart.

Is there already a component or is it to be created?

**Comment by devansh-webkul** [comment]
Hi @ILDaviz,

There is a modal component available in UI package.

File: packages/Webkul/Shop/src/Resources/views/checkout/cart/index.blade.php [hunk 1]
```diff
@@ -128,6 +128,8 @@
                         <div class="misc-controls">
                             <a href="{{ route('shop.home.index') }}" class="link">{{ __('shop::app.checkout.cart.continue-shopping') }}</a>
 
+                            <a href="{{ route('cart.remove.all.items') }}" class="link">{{ __('shop::app.checkout.cart.remove-all-items') }}</a>
```
Comment (jitendra-webkul) [inline packages/Webkul/Shop/src/Resources/views/checkout/cart/index.blade.php:131]: you can use v-alert here like below

<a href="{{ route('cart.remove.all.items') }}" class="link" v-alert>{{ __('shop::app.checkout.cart.remove-all-items') }}</a>

Comment (jitendra-webkul) [reply packages/Webkul/Shop/src/Resources/views/checkout/cart/index.blade.php:131]: This will work only in bliss theme

Comment (ILDaviz) [reply packages/Webkul/Shop/src/Resources/views/checkout/cart/index.blade.php:131]: I'll be finished this week 🙈

Comment (Sanjaybhattwebkul) [reply packages/Webkul/Shop/src/Resources/views/checkout/cart/index.blade.php:131]: > I'll be finished this week see_no_evil

HI @ILDaviz 
Are you **working** for **Remove all cart items at once**?

Comment (ILDaviz) [reply packages/Webkul/Shop/src/Resources/views/checkout/cart/index.blade.php]: yep

Comment (ILDaviz) [reply packages/Webkul/Shop/src/Resources/views/checkout/cart/index.blade.php]: Added alert

File: packages/Webkul/Velocity/src/Routes/front-routes.php [hunk 2]
```diff
@@ -80,6 +80,9 @@
     Route::delete('/cart/remove/{id}', [CartController::class, 'removeProductFromCart'])
         ->name('velocity.cart.remove.product');
 
+    Route::get('/cart/remove-all', [CartController::class, 'removeAllItems'])
```
Comment (devansh-webkul) [inline packages/Webkul/Velocity/src/Routes/front-routes.php:83]: It should not be the GET method and it should be DELETE or POST with CSRF Token.

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

File: packages/Webkul/Velocity/src/Routes/front-routes.php (hunk 2 above)
Comment (ILDaviz) [reply packages/Webkul/Velocity/src/Routes/front-routes.php]: Ok! change method type

Comment (ILDaviz) [reply packages/Webkul/Velocity/src/Routes/front-routes.php]: @devansh-webkul Ok! change method type

**Review by Sanjaybhattwebkul — APPROVED** [review]

**Comment by devansh-webkul** [comment]
Hi @ILDaviz,

Thanks for the contribution :rocket:

**Pull request merged**